docs/quick-start.md

```diff
@@ -0,0 +1,238 @@
+# Getting started with reignite
+
+## Set up networking
+
+```
+# Default network device.
+NET_DEVICE=$(ip route show | awk '/default/ {print $5}')
+
+# Create a macvtap device.
+sudo ip link add link "${NET_DEVICE}" name macvtap0 type macvtap
+
+# Static MAC address can be set,
+# otherwise it gets a random auto-generated address.
+sudo ip link set macvtap0 address 1a:46:0b:ca:bc:7b up
+
+# Check the MAC address of the device.
+ip link show macvtap0
+
+# You can read the MAC address always under /sys without parsing
+# the output of `ip link show`.
+cat /sys/class/net/macvtap0/address
+```
+
+## Containerd
+
+### Create thinpool
+
+Easy quick-start option is to run this script as root. I know,
+it's not recommended in general, and I'm happy you think it's not a good
+way to do things, so there are comments for commands.
+
+```bash
+#!/bin/bash
```

Comment:
What do you think to extracting this script out into `hack/scripts/devpool.sh` or something similar?